webpack.config.js

```diff
@@ -14,6 +14,9 @@ const requiredSentryEnvVar = [
   process.env.SENTRY_URL,
 ]
 
+const assetVariant = process.env.ASSET_VARIANT || 'gov'
+const assetResolveDir = assetVariant === 'edu' ? 'assets/edu/' : 'assets/gov/'
```

Comment:
Is the trailing `/` necessary? if we can drop that without affecting the build the favicon path in HtmlWebpackPlugin would be cleaner-looking